### PR TITLE
Documentation for custom tags

### PR DESCRIPTION
**Pull request by andrew-3**

### CODE REVIEW

File: app/views/pages/developer-guide/integrations/adding-hcaptcha-spam-protection.liquid [hunk 1]
```diff
@@ -332,6 +332,42 @@ document.addEventListener('DOMContentLoaded', function() {
...
+configuration:
+spam_protection:
+  hcaptcha:
+fields:
+  properties:
+    title:
+return_to: /success_page
+---
+
+{% form html-novalidate: true, html-class: 'model-form' %}
```
Comment (piotrze) [inline app/views/pages/developer-guide/integrations/adding-hcaptcha-spam-protection.liquid]: @andrew-3 @diana-lakatos `{% form %}` tag is deprecated. Instead we should provide example with page that have normal html `<form>`. 
Inside <form> you can use `spam_protection` tag or you can enter own code for generating hcaptcha code (https://github.com/Platform-OS/product-marketplace-template/blob/master/app/views/partials/hcaptcha.liquid)
In order to validate hcaptcha you can use this function https://github.com/Platform-OS/product-marketplace-template/blob/master/app/views/partials/lib/validations/hcaptcha.liquid

Please do not add examples that use `{% form %}` tag. We should change all examples that use it.

**Review by piotrze — CHANGES_REQUESTED** [review]
Do not use `{% form %}` tag.

File: app/views/pages/developer-guide/integrations/adding-hcaptcha-spam-protection.liquid (hunk 1 above)
Comment (piotrze) [reply app/views/pages/developer-guide/integrations/adding-hcaptcha-spam-protection.liquid]: @diana-lakatos I've created ticket for it in past, sadly it was closed.....
https://github.com/mdyd-dev/platformos-documentation/issues/1497

**Comment by andrew-3** [comment]
@piotrze I've removed from the page those which were only form-specific, and converted the others from the form tag to HTML tags.

Thanks!

**Pull request merged**